$NetBSD: patch-configure.ac,v 1.2 2015/04/14 03:39:49 mef Exp $

Kill a substitution that /bin/sh on NetBSD objects to.
./configure.lineno: 3557: Syntax error: Bad substitution

--- configure.ac.orig	2014-04-16 19:32:38.000000000 +0000
+++ configure.ac
@@ -34,7 +34,7 @@ then
       # "/c/foo/bar", to simplify later conversions to native Windows
       # format ("c:/foo/bar")
       srcdir=`cd "${srcdir}" && pwd -W`
-      srcdir="/${srcdir:0:1}${srcdir:2}"
+      #srcdir="/${srcdir:0:1}${srcdir:2}"
       ;;
   esac
 fi
